theseus/embodied/kinematics/robot_model.py

```diff
@@ -40,3 +41,34 @@ def forward_kinematics(self, robot_pose: RobotModelInput) -> Dict[str, LieGroup]
 
     def dim(self) -> int:
         return 1
+
+
+class UrdfRobotModel(KinematicsModel):
+    def __init__(self, urdf_path: str):
+        self.drm_model = drm.DifferentiableRobotModel(urdf_path)
+
+    def forward_kinematics(self, joint_states: RobotModelInput) -> Dict[str, LieGroup]:
+        """Computes forward kinematics
+        Args:
+            joint_states: Vector of all joint angles
+        Outputs:
+             Dictionary that maps link name to link pose
+
+        (Note: differentiable robot model uses the xyzw quaternion convention,
+        while theseus uses wxyz, hence the conversion.)
+        """
+        link_poses: Dict[str, LieGroup] = {}
+        for link_name in self.drm_model.get_link_names():
+            pos, quat = self.drm_model.compute_forward_kinematics(
+                joint_states, link_name
```

Comment:
If joint states passed are torch.Tensor then we would need a size check (including appending batch dim if not passed?). Vector initialization should handle check externally so we are good on that.